src/test/fuzz/coins_view.cpp

```diff
@@ -145,10 +146,12 @@ FUZZ_TARGET(coins_view, .init = initialize_coins_view)
                     }
                     auto it{coins_map.emplace(random_out_point, std::move(coins_cache_entry)).first};
                     it->second.AddFlags(flags, *it, sentinel);
+                    usage += it->second.coin.DynamicMemoryUsage();
```

Comment:
this should have been there all along, i.e. you've fixed a small independent accounting bug here, right?

Reply:
Yes!